server/src/main/java/com/lunchtime/service/MyUserDetailsService.java

```diff
@@ -0,0 +1,25 @@
+package com.lunchtime.service;
+
+import com.lunchtime.models.Person;
+import com.lunchtime.repository.PersonRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+
+@Service
+public class MyUserDetailsService implements UserDetailsService {
```

Comment:
Refactored now to MyPersonDetails

Reply:
**Your** person?)

Reply:
yea
now is 
public class MyPersonDetailsServiceImpl implements UserService

Reply:
Why you use prefix My? `PersonDetails` and `PersonService`

Reply:
i so a lot of examples in net and a lot fo them use that kind of naming

Reply:
OK i listen to your porposition  ?

Reply:
prefix service because it is services and prefix detail because there is a search for only two criteria only email and password coming from the database

Reply:
PersonDetailImpl and PersonDetailService

Reply:
ok i will refactor to PersonDetailImpl